compositions/aws-provider/eks/definition.yaml

```diff
@@ -60,6 +62,48 @@ spec:
                   required:
                     - eks-version
                     - eks-subnet-ids
+                managedNodeGroups:
+                  description: Managed Node Groups properties
+                  type: object
+                  properties:
+
+                    mng-minSize:
```

Comment:
prefixing properties with `mng` is unnecessary. These properties are under `managedNodeGroups` property already.  So simply having `minSize`, `desiredSize`, etc would look cleaner IMO